doc/index.rst

```diff
@@ -39,24 +39,28 @@ User Guide
 ==========
 
 The DSM CSC is a support CSC that is used in conjunction with the DSM UI in order
-to produce seeing measurements along the DSM probes line of sight. The functionality to produce these measurements are contained within the UI and are not described in this
-document. The UI is also responsible for the interaction with the DSM probe hardware. 
+to produce seeing measurements along the DSM probes line of sight.
+The functionality to produce these measurements are contained within the UI and are not described in this document.
+The UI is also responsible for the interaction with the DSM probe hardware. 
 The operational interaction of these systems and other details are captured in `SITCOMTN-001 <https://sitcomtn-001.lsst.io/>`_ technical note.
 
-There are two scripts that perform operations of the CSC. They are ``run_dsm.py`` and ``shutdown_dsm.py``. The parameters they take can be found by passing ``-h`` or
-``--help`` to the given script. The ``run_dsm.py`` script constructs the CSC and sends
-it to ``ENABLED`` state, allowing for the CSC to function straight away. It can be
-run in real mode or one of two simulation modes. The simulation modes will be shown
-in the next section. To run the CSC in real mode, do the following.
+There are two scripts that perform operations of the CSC.
+They are ``run_dsm.py`` and ``shutdown_dsm.py``. The parameters they take can be found by passing ``-h`` or ``--help`` to the given script.
+The ``run_dsm.py`` script constructs the CSC and using a flag, sends it to ``ENABLED`` state, allowing for the CSC to function straight away.
+It can be run in real mode or one of two simulation modes.
+The simulation modes will be shown in the next section. To run the CSC in real mode, do the following.
 
 .. prompt:: bash
 
-  run_dsm.py
+  run_dsm.py --state=enabled <index>
+
+The ``<index>`` is an integer value.
+The DSM currently has two indexes (1 and 2) for operational use. 
 
 The ``shutdown_dsm.py`` script can be used to send the CSC to ``STANDBY`` state or
-``OFFLINE`` state. If the CSC is sent to ``OFFLINE``, the process started
-by the ``run_dsm.py`` script will be shutdown and terminated. Since the run script will
-block the current container terminal, you will have to ``docker exec`` into the container to run the shutdown script. For ``STANDBY`` state, run the script this way.
+``OFFLINE`` state.
+If the CSC is sent to ``OFFLINE``, the process started by the ``run_dsm.py`` script will be shutdown and terminated.
+Since the run script will block the current container terminal, you will have to ``docker exec`` into the container to run the shutdown script. For ``STANDBY`` state, run the script this way.
```

Comment:
I confess I am (still) puzzled why you don't use the standard SAL commands to manage this CSC's states, like all other CSCs. It seems a nuisance to have to exec into the Docker container, and as far as I know this is the only CSC that has its own dedicated script to shut it down. There is even a standard set_summary_state script.

Reply:
The extra script is for operator independence as the DSMs are mobile, self-contained units designed to be moved about the facility. The set_summary_script requires access to a script queue which is not always practical from the field. It is unclear to me what kind of priority interrupts are available on the script queue for handling system states while observing is taking place.

Reply:
That's fair. If it's not too late please consider adding something to that effect to this user's guide.

Reply:
Added some text about why the shutdown script is needed.